src/pages/login/[acess].tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from "react";
+
+import { reastorage } from "@reastorage/react";
+import { useRouter } from "next/router";
+
+import { useSignInMutation, AuthType } from "src/graphql";
+import { UserAccessPermissionInfo } from "src/modules/Login/Acess/UserAccessPermissionInfo";
+import { Typography, Box, Button } from "src/ui";
+import { withGraphql } from "src/utils/graphql/withGraphql";
+
+const Acess = () => {
+  const router = useRouter();
+  const { acess, code } = router.query;
+  const [acessMutationResult, acessMutation] = useSignInMutation();
+
+  useEffect(() => {
+    if (!router.isReady || typeof acess !== "string") return;
+    Notification.requestPermission();
+    const pascalMutationType = acess.replace(/^[a-z]/, (char) =>
+      char.toUpperCase(),
+    );
+    if (
+      typeof code !== "string" ||
+      pascalMutationType !== ("Kakao" || "Google" || "Apple")
```

Comment:
Kakao일때를 제외하곤 제대로된 검증이 안될 것 같아요!
```suggestion
      ["Kakao", "Google", "Apple"].every(authProvider => authProvider !== pascalMutationType)
```
이런식으로 작성하는 건 어떨까요?

Reply:
여긴 수정이 안되었는데, 기능 정상동작 여부가 확인 된 걸까요?

Reply:
앗 ! 제가 확인을 못했네요 타입 가드 이용해서 수정하겠습니다 !